keda/templates/metrics-server/apiservice.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     {{- if .Values.certificates.certManager.enabled }}
     {{- if .Values.certificates.certManager.generateCA }}
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Values.operator.name }}-ca
+    {{- else if not .Values.certificates.certManager.issuer.generate }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Values.operator.name }}-tls-certificates
```

Comment:
can't we just unify these? I mean, `{{ .Release.Namespace }}/{{ .Values.operator.name }}-ca` and `{{ .Release.Namespace }}/{{ .Values.operator.name }}-tls-certificates` have the same CA if the issuer is self-generated, so maybe we can just update current condition